crates/hyperdrive-math/Cargo.toml

```diff
@@ -20,6 +15,9 @@ rand = "0.8.5"
 fixed-point = { path = "../fixed-point" }
 fixed-point-macros = { path = "../fixed-point-macros" }
 hyperdrive-wrappers = { path = "../hyperdrive-wrappers" }
+test-utils = { path = "../test-utils" }
+rand_chacha = "0.3.1"
+tracing = "0.1"
```

Comment:
Can we move these to `dev-dependencies`

Reply:
yeah we can now that Agent is only compiled for tests!